cluster/manifests/skipper/hpa.yaml

```diff
@@ -21,3 +21,14 @@ spec:
     resource:
       name: memory
       targetAverageUtilization: {{ .ConfigItems.skipper_ingress_target_average_utilization_memory }}
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 600
+      policies:
+      - type: Pods
+        value: 10
+        periodSeconds: 60
+      - type: Percent
```

Comment:
Not sure why this is needed.

Reply:
to mimic the default behavior and keep it

Reply:
Shouldn't the first policy be enough, even for cases when we scale up by less than 10 pods? Or does it not apply at all, resulting in no scale down?

Reply:
as far as I understood the docs it's either percentage or the value 10, so I want to be safe. Feel free to test to proof that we don't need it.